app/posts/2021-02-01-applications-usability-testing.md

```diff
@@ -0,0 +1,183 @@
+date: 2021-02-01
+related:
+  items:
+  - text: Plaback decks
+  - text: Round 1
+    href: https://docs.google.com/presentation/d/1Q5yzgXKDwzCO9MwH2bZkKXlCCqUQqk7aj1lkfuY_ATg/edit#slide=id.p
+  - text: Round 2
+    href: https://docs.google.com/presentation/d/124pKthA6yIkvYhwKnx7uK9_dCfnEuIOPyZmUp3dUagk/edit#slide=id.p
+  - text: Round 3
+    href: https://docs.google.com/presentation/d/1OWWx2lWNdLW09es7XTRzENJCpDCryggFoV9giDkHahw/edit#slide=id.p
+  - text: Miro boards
+  - text: Applications release 1 TA
+    href: https://drive.google.com/drive/u/0/folders/13uar7gmJOR7x2Uz1kQq78eAE1voeDCp0
+  - text: S63 - HMW Application (Part 1)
+    href: https://drive.google.com/drive/u/0/folders/1nvRE8HH7okFeLWdF6uLORCdTt3nBkEMp
+  - text: S63 - HMW Application (Part 2)
+    href: https://drive.google.com/drive/u/0/folders/1nvRE8HH7okFeLWdF6uLORCdTt3nBkEMp
+  - text: Applications research scope (oct 20)
+    href: https://drive.google.com/drive/u/0/folders/1pVyH666DOZCbqhn8GunQKaWMclu3u5f1
+  - text: Application research (analysis) Round 2
+    href: https://drive.google.com/drive/u/0/folders/1wbtXUdQqZyP3Q8kxRZJkt1j_57V3Pe9X
+  - text: S74 - Support Workshop
+    href: https://drive.google.com/drive/u/0/folders/1wbtXUdQqZyP3Q8kxRZJkt1j_57V3Pe9X
+  - text: Applications release 1 & measures
+    href: https://drive.google.com/drive/u/0/folders/1JHpUO0NwT45yL5RQXQXVq_bGuWRVStqB
+  - text: Applications release 1 story map
+    href: https://drive.google.com/drive/u/0/folders/1JHpUO0NwT45yL5RQXQXVq_bGuWRVStqB
+  - text: A - JS Applications release 1
+    href: https://drive.google.com/drive/u/0/folders/1JHpUO0NwT45yL5RQXQXVq_bGuWRVStqB
+  - text: S77 - Application Round 3 Research(JS)
+    href: https://drive.google.com/drive/u/0/folders/1GmpFWxLPFEcsajQpzrGPjfp2wKI4WqtO
+  - text: Download application
+    href: https://drive.google.com/drive/u/0/folders/1GmpFWxLPFEcsajQpzrGPjfp2wKI4WqtO
+  - text: TVS notifications
+    href: https://drive.google.com/drive/u/0/folders/1q9pRP3DTxFjRKLyFwbvz0UYMK83maPjB
+  - text: Application ideation
+    href: https://drive.google.com/drive/u/0/folders/1lnhaX_Q4-h4g3ec_Hsk7Y2HROWlQAOtr
+---
+
+The discovery research conducted in October 2020 highlighted the benefits of providing a standardised application form for teachers to apply for jobs listed in Teaching vacancies. Having a single application format would make it easier for jobseekers to apply for those jobs as they wouldn’t have to fill applications that came not only in a variety of formats but also asked different questions, which are current pain points for teachers. 
+
+Given the above, a basic Google forms application was created to explore the landscape with teachers and from there a more detailed prototype was developed and tested, giving rise to a second prototype which was also tested with teachers during round 3, in order to refine the application process from the jobseeker’s point of view. 
```

Comment:
This might be personal preference but I would look to split this into two sentences, something like

Given the above, a basic Google forms application was created to explore the landscape with teachers and from there a more detailed prototype was developed and tested. This led to a second prototype which was also tested with teachers during round 3, in order to refine the application process from the jobseeker’s point of view.